src/admin.rs

```diff
@@ -52,6 +57,14 @@ pub struct ConfigDump {
     static_config: LocalConfig,
     version: BuildInfo,
     config: Config,
+    certificates: Vec<CertsDump>,
+}
+
+#[derive(serde::Serialize, Debug, Clone)]
+pub struct CertsDump {
+    identity: String,
+    leaf: String,
+    chain: Vec<String>,
```

Comment:
would be good to include [not before/after timestamps](https://github.com/istio/ztunnel/blob/master/src/tls/boring.rs#L115-L116).  I have seen these go expired before while debugging customer env's - it was nice to have that information front and center